### PR TITLE
Add `id-card`

### LINKED ISSUE #953: ID card icon
## Icon Request

* Icon name: identification
* Use case: ID, passport, verification, KYC
* _Screenshots_ of similar icons: 
Screenshot_3
Screenshot_4
Screenshot_1

### PR DESCRIPTION
**Pull request by jguddas**
Resolves #953

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Maybe let's call this `id-badge` instead, although I'm not quite happy about the user shape still.

**Comment by karsa-mistmere** [comment]
I still definitely want to make this work within the bounds of credit card, to be consistent:
image
image

I feel like taking into consideration discussions in #609, these should be able to work.

**Comment by ericfennis** [comment]
@karsa-mistmere Yeah good point. I like the second one, can we increase the length of the text lines a bit?

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere Yeah good point. I like the second one, can we increase the length of the text lines a bit?

Sure: id-card

**Comment by ericfennis** [comment]
LGTM @karsa-mistmere Go with that one?

**Comment by jguddas** [comment]
Now we have a 1px gap violation between the head and the body.

**Comment by karsa-mistmere** [comment]
> Now we have a 1px gap violation between the head and the body.

Yes, but see https://github.com/lucide-icons/lucide/issues/609#issuecomment-1203678217

File: icons/id-card.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="8" cy="10" r="1" />
+  <rect x="2" y="5" width="20" height="14" rx="2" />
+  <path d="M14 14h4" />
+  <path d="M6 15a2.5 2.5 0 0 1 4 0" />
```
Comment (jguddas) [inline icons/id-card.svg:15]:
```suggestion
  <path d="M6 15h4" />
```

**Comment by jguddas** [comment]
> > Now we have a 1px gap violation between the head and the body.
> 
> Yes, but see [#609 (comment)](https://github.com/lucide-icons/lucide/issues/609#issuecomment-1203678217)

The violation is not really worth it, the whole icon hasn't been that hugely requested, and I'm fine with just closing this PR as well.

File: icons/id-card.svg (hunk 1 above)
Comment (karsa-mistmere) [reply icons/id-card.svg:15]: Not a fan, that's exactly the issue we were trying to avoid ("head looks detached from the body").

Comment (jguddas) [reply icons/id-card.svg:15]: I really dislike it as well, I just generally don't like our human icons, maybe we can get some sort of animal related outline there instead?

Comment (karsa-mistmere) [reply icons/id-card.svg:15]: Perfect! image

Comment (jguddas) [reply icons/id-card.svg:15]: Best I could come up with:

image

Comment (karsa-mistmere) [reply icons/id-card.svg:15]: How about just adding a silhouette like this?
image

Comment (jguddas) [reply icons/id-card.svg:15]: Like a playing figure, that's not a bad idea.

**Comment by ericfennis** [comment]
I think we could make an exception for this one to accept this small violation. 
Let's find a balance between making it recognizable, and consistent and between our guidelines. But with some, we could make exceptions. This icon has a valid usecase and is valuable in modern applications. It would be a shame if we couldn't do this because of a small violation.

I'm curious what you think of this.

**Comment by jguddas** [comment]
New idea: icon - 2024-03-28T222422 505

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M+6+9+L+9+9%22+/%3E%3Cpath+d=%22M21.727+18A2+2+0+0+1+20+19H4a2+2+0+0+1-2-2V7a2+2+0+0+1+2-2h8%22+/%3E%3Cpath+d=%22M22+14.333a4+4+0+0+0-8+0%22+/%3E%3Cpath+d=%22M6+13h4%22+/%3E%3Ccircle+cx=%2218%22+cy=%227.667%22+r=%222.667%22+/%3E">icons<br/>Open lucide studio</a>

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — COMMENTED** [review]
Hmm, still not convinced with the design, to me looks a bit weird that the user icon is out of the boundaries of the card.. 🤔

**Comment by karsa-mistmere** [comment]
> Hmm, still not convinced with the design, to me looks a bit weird that the user icon is out of the boundaries of the card.. 🤔

Yeah, I kinda agree. How about this one? It's maybe a bit dense, but not that bad, I guess, we definitely have worse icons by density.

image

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M16+10h2%22+/%3E%3Cpath+d=%22M16+14h2%22+/%3E%3Cpath+d=%22M6.17+15a3+3+0+0+1+5.66+0%22+/%3E%3Ccircle+cx=%229%22+cy=%2211%22+r=%222%22+/%3E%3Crect+x=%222%22+y=%225%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image">icons<br/>Open lucide studio</a>

**Comment by karsa-mistmere** [comment]
Alternatively:

image

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M13+19a4+4+0+0+0-8+0%22+/%3E%3Cpath+d=%22M16+10h2%22+/%3E%3Cpath+d=%22M16+14h2%22+/%3E%3Ccircle+cx=%229%22+cy=%2212%22+r=%223%22+/%3E%3Crect+x=%222%22+y=%225%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image">icons<br/>Open lucide studio</a>

**Comment by jguddas** [comment]
> Alternatively:
> 
> image
> 
> [iconsOpen lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M13+19a4+4+0+0+0-8+0%22+/%3E%3Cpath+d=%22M16+10h2%22+/%3E%3Cpath+d=%22M16+14h2%22+/%3E%3Ccircle+cx=%229%22+cy=%2212%22+r=%223%22+/%3E%3Crect+x=%222%22+y=%225%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image)

I like this one, the first one not so much, the body has a bit of a weird gap on the lower left tho.

image

**Comment by jguddas** [comment]
> Like Eric, I prefer the first one:
> 
> > image
> > [iconsOpen lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M16+10h2%22+/%3E%3Cpath+d=%22M16+14h2%22+/%3E%3Cpath+d=%22M6.17+15a3+3+0+0+1+5.66+0%22+/%3E%3Ccircle+cx=%229%22+cy=%2211%22+r=%222%22+/%3E%3Crect+x=%222%22+y=%225%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image)

Feels to dense for me.

**Comment by karsa-mistmere** [comment]
> > Like Eric, I prefer the first one:
> > > image
> > > [iconsOpen lucide studio](https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M16+10h2%22+/%3E%3Cpath+d=%22M16+14h2%22+/%3E%3Cpath+d=%22M6.17+15a3+3+0+0+1+5.66+0%22+/%3E%3Ccircle+cx=%229%22+cy=%2211%22+r=%222%22+/%3E%3Crect+x=%222%22+y=%225%22+width=%2220%22+height=%2214%22+rx=%222%22+/%3E&name=image)
> 
> Feels to dense for me.

A bit too dense for my liking as well, but I think it's still better than any alternatives so far and it's not extremely dense by any means.

**Comment by gyurielf** [comment]
Hello there!

Is there any chance to merge this ?
I would definitely fine with the latest version :)

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**